### PR TITLE
Added Additional Info About Integration Managing

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added Additional Info About Integration Managing. Changed outdated screenshot.

This pull request (PR) ...

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- ...

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- ...

## Affected documentation pages

https://docs.magento.com/user-guide/system/integrations.html

whatsnew
Updated [Integrations](https://docs.magento.com/user-guide/system/integrations.html) with new information for activating and deleting an integration.

### CODE REVIEW

File: src/system/integrations.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ _Integrations_
 
    - Enter the **Identity Link URL** to redirect the users to a third-party account with these Magento integration credentials.
 
+   - Enter the current Admin User password into **Your Password** field.
```
Comment (leslietilling) [inline src/system/integrations.md:28]:
```suggestion
   - When prompted, enter **Your Password** to confirm your identity.
```

File: src/system/integrations.md [hunk 2]
```diff
@@ -42,6 +44,23 @@ _Integrations_
 
 1. When complete, click **Save**.
 
+## Activating an integration
```
Comment (leslietilling) [inline src/system/integrations.md:47]:
```suggestion
## Activate an integration
```

File: src/system/integrations.md [hunk 3]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+By default, when the integration was saved, it appears on the grid in _Inactive_ status. To activate it, perform the following steps:
```
Comment (leslietilling) [inline src/system/integrations.md:49]:
```suggestion
By default, a saved integration appears on the grid with an _Inactive_ status. To activate it, complete the following steps:
```

File: src/system/integrations.md [hunk 4]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+1. Find created integration on grid **System** > _Extensions_ > **Integrations**.
```
Comment (leslietilling) [inline src/system/integrations.md:51]:
```suggestion
1. On the _Admin_ sidebar, go to **System** > _Extensions_ > **Integrations**.
```

File: src/system/integrations.md [hunk 5]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+1. Click the **Activate** link.
```
Comment (leslietilling) [inline src/system/integrations.md:53]:
```suggestion
1. Find the newly created integration and click the **Activate** link.
```

File: src/system/integrations.md [hunk 6]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+1. In the upper-right corner click the **Allow** button.
```
Comment (leslietilling) [inline src/system/integrations.md:55]:
```suggestion
1. In the upper-right corner, click <span class="btn">Allow</span>.
```

File: src/system/integrations.md [hunk 7]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+   Integration Tokens for Extensions will be appeared:
```
Comment (leslietilling) [inline src/system/integrations.md:57]:
```suggestion
   This displays the Integration Tokens for Extensions. Copy this information to a secure, encrypted location for use with your integration.
```

File: src/system/integrations.md [hunk 8]
```diff
@@ -42,6 +44,23 @@ _Integrations_
...
+
+   ![Integration Tokens for Extensions]({% link images/images/integration-tokens-for-extensions.png %}){: .zoom}
+   _Integration Tokens for Extensions_
+
+1. In the upper-right corner click the **Done** button.
```
Comment (leslietilling) [inline src/system/integrations.md:62]:
```suggestion
1. In the upper-right corner, click <span class="btn">Done</span>.
```

File: src/system/integrations.md [hunk 9]
```diff
@@ -61,5 +80,13 @@ By default, the system does not permit anonymous guest access to CMS, catalog, a
 
 See [Restricting access to anonymous web APIs][2]{:target="_blank"} in the Magento developer documentation.
 
+## Deleting an Integration
```
Comment (leslietilling) [inline src/system/integrations.md:83]:
```suggestion
## Delete an integration
```

File: src/system/integrations.md [hunk 10]
```diff
@@ -61,5 +80,13 @@ By default, the system does not permit anonymous guest access to CMS, catalog, a
...
+
+1. Find created integration on grid **System** > _Extensions_ > **Integrations**.
```
Comment (leslietilling) [inline src/system/integrations.md:85]:
```suggestion
1. On the _Admin_ sidebar, go to **System** > _Extensions_ > **Integrations**.
```

File: src/system/integrations.md [hunk 11]
```diff
@@ -61,5 +80,13 @@ By default, the system does not permit anonymous guest access to CMS, catalog, a
...
+
+1. Click the delete icon in **Delete** column
```
Comment (leslietilling) [inline src/system/integrations.md:87]:
```suggestion
1. Find the existing integration and click the icon ( ![trashcan icon]({% link images/images/btn-trashcan2.png %}) ) in the **Delete** column.
```

File: src/system/integrations.md [hunk 12]
```diff
@@ -61,5 +80,13 @@ By default, the system does not permit anonymous guest access to CMS, catalog, a
...
+
+1. Confirm your action by clicking **OK**
```
Comment (leslietilling) [inline src/system/integrations.md:89]:
```suggestion
1. To confirm your action, click <span class="btn">OK</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

File: src/system/integrations.md [hunk 13]
```diff
@@ -61,5 +80,13 @@ By default, the system does not permit anonymous guest access to CMS, catalog, a
 
 See [Restricting access to anonymous web APIs][2]{:target="_blank"} in the Magento developer documentation.
 
+## Delete an Integration
```
Comment (leslietilling) [inline src/system/integrations.md:83]:
```suggestion
## Delete an integration
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**